sycl/include/sycl/handler.hpp

```diff
@@ -2870,6 +2893,25 @@ class __SYCL_EXPORT handler {
   void memcpyFromDeviceGlobal(void *Dest, const void *DeviceGlobalPtr,
                               bool IsDeviceImageScoped, size_t NumBytes,
                               size_t Offset);
+
+  template <typename T, int Dims, access::mode AccessMode,
+            access::target AccessTarget,
+            access::placeholder IsPlaceholder = access::placeholder::false_t,
+            typename PropertyListT = property_list>
+  void checkIfPlaceholderIsBoundedToCG(
```

Comment:
https://github.com/intel/llvm/pull/8131 introduced a way to check whether a placeholder is bound to a CG. It might be good to unify all the checks.

Also, a nit: I _think_ it's `bound` rather than `bounded`.

Reply:
I've investigated a little and I guess It's not possible because there are two different approaches:
- here we have an accessor, which is passed as an argument to a command (`fill()`, `copy()`, `update_host()`), so we need to check that this certain accessor is bound to the handler.
- in finalize() (https://github.com/intel/llvm/pull/8131) function we're checking that the number of really used accessors matches the number of requirements.
So, in the first case there is no need to check that the numbers are matching, in the second one - we don't know if the accessor is really used, so we can't be sure that it should be in `MAssociatedAccesors`.

If you have any idea about this unification - feel free to suggest.